src/elements/ts-main/ts-dashboard.html

```diff
@@ -174,10 +254,55 @@
           </template>
       </paper-dialog>
 
+      <paper-dialog id="confirm" class="popup" modal="true" entry-animation="scale-up-animation" exit-animation="scale-down-animation">
+          <h2>Overwrite?</h2>
+          <div>
+              <p>This file already exists. Overwrite?</p>
+          </div>
+          <div id="confirmbuttons">
+              <paper-button dialog-dismiss>No</paper-button>
+              <paper-button on-tap="executebackup">Confirm</paper-button>
+          </div>
+      </paper-dialog>
+
+      <paper-dialog id="import-dialog" class="popup" modal="true" entry-animation="scale-up-animation" exit-animation="scale-down-animation">
+          <h2>Choose Translation</h2>
+          <div>
+              <template is="dom-if" if="{{projectlist.length}}">
```

Comment:
This should be a list of projects not target translations. The terminology in the code is not currently correct so it can be miss-leading. This is the same list of projects a user sees when they are starting a new translation.